epochStart/mock/messengerStub.go

```diff
@@ -0,0 +1,157 @@
+package mock
+
+import (
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+// MessengerStub -
+type MessengerStub struct {
+	ConnectedPeersCalled func() []p2p.PeerID
```

Comment:
If you have made an interface in epochStart like
```GO
interface P2PMessenger struct{
     ConnectedPeers() []p2p.PeerID
     IsInterfaceNil() bool
}
```
and have used this interface instead, you would have improved the decoupling of this package and this stub would have been shorter.

Reply:
done. made a larger interface so I can use it also in process.go.